src/docs/components/tabs/tab.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+	import { getTabsContext } from '$docs/components/tabs/root.svelte';
+	import { Npm, Yarn, Pnpm } from '$docs/components/icons';
+
+	export let tab: string;
+
+	const { trigger } = getTabsContext();
+</script>
+
+<button
+	{...$trigger(tab)}
+	use:trigger
+	class="rounded-md border border-transparent bg-neutral-800 px-3 py-2 text-neutral-400 transition
+        hover:opacity-100 focus:!border-magnum-400 focus:!text-magnum-400
+        data-[state=active]:border-magnum-700 data-[state=active]:py-2 data-[state=active]:text-magnum-600 data-[state=active]:opacity-100"
+>
+	<div class="flex items-center gap-2 px-1">
+		{#if tab === 'npm'}
```

Comment:
Didn't we have a Tabs component with NPM stuff already?

Reply:
We did, it's just moved to a different directory outside of the routes!